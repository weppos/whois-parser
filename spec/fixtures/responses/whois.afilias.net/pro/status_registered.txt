Domain Name: GOOGLE.PRO
Registry Domain ID: D107300000000011545-LRMS
Registrar WHOIS Server: whois.markmonitor.com
Registrar URL: http://www.markmonitor.com
Updated Date: 2017-08-07T09:24:28Z
Creation Date: 2008-07-22T00:00:00Z
Registry Expiry Date: 2018-09-08T00:00:00Z
Registrar Registration Expiration Date:
Registrar: MarkMonitor Inc.
Registrar IANA ID: 292
Registrar Abuse Contact Email: abusecomplaints@markmonitor.com
Registrar Abuse Contact Phone: +1.2083895740
Reseller:
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Registry Registrant ID: C51599151-LRMS
Registrant Name: DNS Admin
Registrant Organization: Google Inc.
Registrant Street: 1600 Amphitheatre Parkway
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6502530000
Registrant Phone Ext:
Registrant Fax: +1.6506188571
Registrant Fax Ext:
Registrant Email: dns-admin@google.com
Registry Admin ID: C51599151-LRMS
Admin Name: DNS Admin
Admin Organization: Google Inc.
Admin Street: 1600 Amphitheatre Parkway
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6502530000
Admin Phone Ext:
Admin Fax: +1.6506188571
Admin Fax Ext:
Admin Email: dns-admin@google.com
Registry Tech ID: C51599151-LRMS
Tech Name: DNS Admin
Tech Organization: Google Inc.
Tech Street: 1600 Amphitheatre Parkway
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6502530000
Tech Phone Ext:
Tech Fax: +1.6506188571
Tech Fax Ext:
Tech Email: dns-admin@google.com
Registry Billing ID: C166371517-LRMS
Billing Name: CCOPS Billing
Billing Organization: MarkMonitor, Inc.
Billing Street: 391 N. Ancestor Place
Billing City: Boise
Billing State/Province: ID
Billing Postal Code: 83704
Billing Country: US
Billing Phone: +1.2083895740
Billing Phone Ext:
Billing Fax: +1.2083895771
Billing Fax Ext:
Billing Email: ccopsbilling@markmonitor.com
Name Server: NS1.GOOGLE.COM
Name Server: NS2.GOOGLE.COM
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2018-05-04T12:52:18Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to AFILIAS WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the Afilias registry database. The data in this record is provided by Afilias Limited for informational purposes only, and Afilias does not guarantee its accuracy.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to(a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Afilias reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
