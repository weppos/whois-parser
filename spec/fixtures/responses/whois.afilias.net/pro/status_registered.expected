#disclaimer
  %s == "Access to AFILIAS WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the Afilias registry database. The data in this record is provided by Afilias Limited for informational purposes only, and Afilias does not guarantee its accuracy.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to(a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Afilias reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy."


#domain
  %s == "google.pro"

#domain_id
  %s == "D107300000000011545-LRMS"


#status
  %s == ["clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited", "clientTransferProhibited https://icann.org/epp#clientTransferProhibited", "clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2008-07-22 00:00:00 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-08-07 09:24:28 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-09-08 00:00:00 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "292"
  %s.name         == "MarkMonitor Inc."
  %s.organization == "MarkMonitor Inc."
  %s.url          == "http://www.markmonitor.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "C51599151-LRMS"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6506188571"
  %s[0].email         == "dns-admin@google.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "C51599151-LRMS"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6506188571"
  %s[0].email         == "dns-admin@google.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "C51599151-LRMS"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6506188571"
  %s[0].email         == "dns-admin@google.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
